CMakeLists.txt

```diff
@@ -24,61 +27,20 @@ endif ()
 
 find_package(MPI REQUIRED)
 
-# Add include paths
-if (FIBER_ENABLE_HEFFTE)
-add_definitions(-DFIBER_ENABLE_HEFFTE)
-include_directories($ENV{HOME}/heffte/build/include)
-link_directories($ENV{HOME}/heffte/build)
-endif()
-
-if (FIBER_ENABLE_FFTMPI)
-add_definitions(-DFIBER_ENABLE_FFTMPI)
-include_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-link_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-endif()
-
-if (FIBER_ENABLE_ACCFFT)
-add_definitions(-DFIBER_ENABLE_ACCFFT)
```

Comment:
`add_definitions()` adds a compile-time flag. Now the flag is missing. Is this intended?